Source/Core/DolphinWX/LuaScripting.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <map>
+#include <math.h>
+#include <mutex>
+#include <wx/artprov.h>

```

Comment:
I was able to replace every include with a forward declarations with the exceptions of <wx/frame.h> and <map>. I'm pretty sure frame.h must be included, but I'm not so sure about map. Logically it seems like I should be able to because all I do in this file is declare that the map exists, but for some reason I keep getting errors when I try to forward declare it, idk.